### PR TITLE
This is a pull request

### PR DESCRIPTION
**Pull request by guyws**
review away

### CODE REVIEW

File: src/main/java/frc/team3388/robot/RobotMap.java [hunk 1]
```diff
@@ -10,4 +10,11 @@
 
     // HID
     public static final HidChannel XBOX = RoboRio.newHidChannel(0);
+
+    public static final int INTAKE_SYSTEM_MOTOR = 1;
+
```
Comment (tomtzook) [inline src/main/java/frc/team3388/robot/RobotMap.java]: Remove double spaces between the constants.

File: src/main/java/frc/team3388/robot/SystemFactory.java [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 import com.flash3388.flashlib.frc.robot.FrcRobotControl;
+import com.flash3388.flashlib.frc.robot.io.devices.FrcDoubleSolenoid;
+import com.flash3388.flashlib.frc.robot.io.devices.FrcSpeedController;
 import com.flash3388.flashlib.frc.robot.io.devices.SpeedControllers;
+import com.flash3388.flashlib.io.devices.DoubleSolenoid;
 import com.flash3388.flashlib.io.devices.SpeedController;
 import frc.team3388.robot.subsystems.ExampleSystem;
+import frc.team3388.robot.subsystems.IntakeSystem;
+
+import java.awt.Robot;
```
Comment (tomtzook) [inline src/main/java/frc/team3388/robot/SystemFactory.java]: Hmm what is this import? It's unused anyhow so remove.

You can do `ctrl+o` in Intellij to remove unused imports in a file.

File: src/main/java/frc/team3388/robot/subsystems/IntakeSystem.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import com.flash3388.flashlib.scheduling.Subsystem;
+import edu.wpi.first.wpilibj.Relay;
+
+public class IntakeSystem extends Subsystem {
+
+    private SpeedController motor;
+
+    private DoubleSolenoid pistons;
+
+    private final double speed = 0.6;
```
Comment (tomtzook) [inline src/main/java/frc/team3388/robot/subsystems/IntakeSystem.java]: This can be a constant. i.e. 

```java
private static final double SPEED = 0.6
```

**Review by tomtzook — CHANGES_REQUESTED** [review]
Nice work generally.

You forgot to add code to the robot class which creates the system.
Take a look at the example system.

File: src/main/java/frc/team3388/robot/RobotMap.java [hunk 4]
```diff
@@ -7,7 +7,11 @@
 
     // EXAMPLE SYSTEM
     public static final int EXAMPLE_SYSTEM_MOTOR = 5;
-
     // HID
     public static final HidChannel XBOX = RoboRio.newHidChannel(0);
+    public static final int INTAKE_SYSTEM_MOTOR = 1;
+    public static final int PISTON_REVERSE_CHANNEL = 2;
+    public static final int PISTON_FORWARD_CHANNEL = 3;
+    public static final double INTAKE_SPEED = 0.6;
```
Comment (tomtzook) [inline src/main/java/frc/team3388/robot/RobotMap.java]: `RobotMap` is only for connections. This is not.

Put this in the system class.

File: src/main/java/frc/team3388/robot/subsystems/IntakeSystem.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    private DoubleSolenoid pistons;
+
+
+
+    public IntakeSystem(SpeedController motor, DoubleSolenoid pistons) {
+        this.motor = motor;
+        this.pistons = pistons;
+    }
+
+    public void collect(double speed) {
```
Comment (tomtzook) [inline src/main/java/frc/team3388/robot/subsystems/IntakeSystem.java]: You should not receive a parameter. This system is meant to only move at a specific speed. What you had earlier was the intention.

**Review by tomtzook — CHANGES_REQUESTED** [review]
Hmm... not really what I mean.

Java constants can be defined in any class, btw.

File: src/main/java/frc/team3388/robot/RobotMap.java (hunk 4 above)
Comment (guyws) [reply src/main/java/frc/team3388/robot/RobotMap.java]: ok

File: src/main/java/frc/team3388/robot/subsystems/IntakeSystem.java (hunk 5 above)
Comment (guyws) [reply src/main/java/frc/team3388/robot/subsystems/IntakeSystem.java]: yeah thats related to the other thing

**Review by tomtzook — APPROVED** [review]
Nice work

**Pull request merged**